### DISCUSSION TITLE
[Question][Hw2]

### DISCUSSION THREAD

**Discussion by Jaegoomon** [prosyslab-classroom/cs348-information-security#503] [Q&A]
Hello,

![image](https://user-images.githubusercontent.com/40551890/114333514-6e63fe00-9b83-11eb-8672-9a34a2b30ec6.png)

This is my result of attack in mongo DB, in this picture as you can see, `resbytes` is both zero. 

I have no idea why resbytes is zero, which factors can change those attributes? I just send a lot of request to amp server. 
It seems that amp server get the packet well, 

Is there anyone who advise for me? plz....

**Comment by robotjoaa** [comment]
It seems you are trying to send NTP request packets for amplification attack. Were you able to get the response properly if you set the source IP as the attacker not victim? I'm also suffering from the similar issue, so I would like to check.

**Comment by gyusss** [comment]
Dooray! 메일 발송 실패 안내



  
  
    
    
  
  
    
메일 발송
실패 안내
    
  
  
    
        
          
***@***.***) 
          님께
보낸
메일이
전송되지
못하였습니다.
        
        
          실패 사유를 확인해보세요.
        
    
  
  
    
      
        
          
            * 받는 사람 : 
***@***.***) 
          
          
            * 발송 시간 : 
2021-04-12T20:37:12
          
          
            * 메일 제목 : 
Re: [prosyslab-classroom/cs448-2021-spring] [Question][Hw2] (#35)
          
          
            
              
                * 실패 사유 : 
              
              
                
받는 사람이 회원님의 메일을 수신차단 하였습니다.
                
              
            
                      
          
        
      
    
  
  
    
        
          이 메일은 발신전용으로 회신되지 않습니다.
          더 궁금하신 사항은
          ***@***.***
          으로 문의해 주시기 바랍니다.
        
    
  
  
    
        © Dooray!.

**Comment by Jaegoomon** [comment]
@robotjoaa 
you mean that I'd better change the src address(victim) into attacker's for checking response?

**Comment by gyusss** [comment]
Dooray! 메일 발송 실패 안내



  
  
    
    
  
  
    
메일 발송
실패 안내
    
  
  
    
        
          
***@***.***) 
          님께
보낸
메일이
전송되지
못하였습니다.
        
        
          실패 사유를 확인해보세요.
        
    
  
  
    
      
        
          
            * 받는 사람 : 
***@***.***) 
          
          
            * 발송 시간 : 
2021-04-12T22:09:35
          
          
            * 메일 제목 : 
Re: [prosyslab-classroom/cs448-2021-spring] [Question][Hw2] (#35)
          
          
            
              
                * 실패 사유 : 
              
              
                
받는 사람이 회원님의 메일을 수신차단 하였습니다.
                
              
            
                      
          
        
      
    
  
  
    
        
          이 메일은 발신전용으로 회신되지 않습니다.
          더 궁금하신 사항은
          ***@***.***
          으로 문의해 주시기 바랍니다.
        
    
  
  
    
        © Dooray!.

**Comment by robotjoaa** [comment]
> @robotjoaa
> you mean that I'd better change the src address(victim) into attacker's for checking response?

Right. I just wanted to check whether you've got proper response from the amp server for that request packet at least once.

**Comment by gyusss** [comment]
Dooray! 메일 발송 실패 안내



  
  
    
    
  
  
    
메일 발송
실패 안내
    
  
  
    
        
          
***@***.***) 
          님께
보낸
메일이
전송되지
못하였습니다.
        
        
          실패 사유를 확인해보세요.
        
    
  
  
    
      
        
          
            * 받는 사람 : 
***@***.***) 
          
          
            * 발송 시간 : 
2021-04-12T23:57:14
          
          
            * 메일 제목 : 
Re: [prosyslab-classroom/cs448-2021-spring] [Question][Hw2] (#35)
          
          
            
              
                * 실패 사유 : 
              
              
                
받는 사람이 회원님의 메일을 수신차단 하였습니다.
                
              
            
                      
          
        
      
    
  
  
    
        
          이 메일은 발신전용으로 회신되지 않습니다.
          더 궁금하신 사항은
          ***@***.***
          으로 문의해 주시기 바랍니다.
        
    
  
  
    
        © Dooray!.

**Comment by Jaegoomon** [comment]
I understand what you mean. Thank you! I'll try that first

**Comment by gyusss** [comment]
Dooray! 메일 발송 실패 안내



  
  
    
    
  
  
    
메일 발송
실패 안내
    
  
  
    
        
          
***@***.***) 
          님께
보낸
메일이
전송되지
못하였습니다.
        
        
          실패 사유를 확인해보세요.
        
    
  
  
    
      
        
          
            * 받는 사람 : 
***@***.***) 
          
          
            * 발송 시간 : 
2021-04-13T01:36:03
          
          
            * 메일 제목 : 
Re: [prosyslab-classroom/cs448-2021-spring] [Question][Hw2] (#35)
          
          
            
              
                * 실패 사유 : 
              
              
                
받는 사람이 회원님의 메일을 수신차단 하였습니다.
                
              
            
                      
          
        
      
    
  
  
    
        
          이 메일은 발신전용으로 회신되지 않습니다.
          더 궁금하신 사항은
          ***@***.***
          으로 문의해 주시기 바랍니다.
        
    
  
  
    
        © Dooray!.
